libraries/bezier-rs/src/utils.rs

```diff
@@ -131,6 +129,7 @@ pub fn solve_cubic(a: f64, b: f64, c: f64, d: f64) -> Vec<f64> {
 		} else {
 			// If a is approximately 0, treat as a quadratic problem
 			let discriminant = c * c - 4. * b * d;
+			println!("{:?}", discriminant);
```

Comment:
Remember to remove println, which I assume is left over.

Reply:
Ah, thanks for catching that!